frontend/src/components/InputText.js

```diff
@@ -1,8 +1,73 @@
 import React, {Component} from 'react';
+import InputTextItems from './InputTextItems'
+import './InputText.css'
 
 class InputText extends Component {
+    constructor(properties) {
+        super(properties);
+       
+        this.state = {
+          items: []
+        };
+       
+        this.addItem = this.addItem.bind(this);
+        this.deleteItem = this.deleteItem.bind(this);
+
+    }
+    addItem(event) {
```

Comment:
Add space between line 16 and 17, same goes for all ending and beginnings of methods